### PR TITLE
release 0.2.91

### PR DESCRIPTION
**Pull request by danking**

### CODE REVIEW

**Comment by danking** [comment]
wait for https://github.com/hail-is/hail/pull/11614#pullrequestreview-914351359

File: hail/python/hail/docs/change_log.md [hunk 1]
```diff
@@ -23,6 +23,17 @@ relating to file formats**: this means that it may not be possible to use
 an earlier version of Hail to read files written in a later version.
 
 
+---
+
+## Version 0.2.91
+
+Release 2022-03-18
+
+### Bug fixes
```
Comment (johnc1231) [inline hail/python/hail/docs/change_log.md:32]: https://github.com/hail-is/hail/pull/11563 was also a bug fix

**Review by johnc1231 — CHANGES_REQUESTED** [review]

File: hail/python/hail/docs/change_log.md (hunk 1 above)
Comment (johnc1231) [reply hail/python/hail/docs/change_log.md:32]: And depending on how desperately the Leo people need a release, I'd like to get #11611 in this as well

**Comment by danking** [comment]
wait also for #11611

**Review by johnc1231 — APPROVED** [review]

**Pull request merged**